clientlib-java/src/main/java/org/daisy/pipeline/client/models/Argument.java

```diff
@@ -135,7 +132,6 @@ private void lazyLoad() {
 				}
 
 				this.defaultValue = XPath.selectText("@default", argumentNode, XPath.dp2ns);
-				this.dataType = XPath.selectText("@data-type", argumentNode, XPath.dp2ns);
```

Comment:
@bertfrees Will there be backwards compatibility issues? Could we still check for the `@data-type` just in case the engine is not the latest version?

Reply:
Yes it will be backwards-incompatible. I was hoping we wouldn't have to deal with those issues. As a general solution I think we can check the version of the server and show a connection error in the web UI if it's incompatible.

But in this case I guess we could indeed easily check `@data-type` and let it override `@type`.